Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -169,7 +169,8 @@ def GetDscLibrarys (self, BuildPkgName = "BootLoaderCorePkg"):
             'FlashDescriptorLib|Silicon/ApollolakePkg/Library/FlashDescriptorLib/FlashDescriptorLib.inf',
             'VtdLib|Silicon/$(SILICON_PKG_NAME)/Library/VtdLib/VtdLib.inf',
             'SmbusLib|Silicon/$(SILICON_PKG_NAME)/Library/SmbusLib/SmbusLib.inf',
-            'HdaLib|Platform/$(BOARD_PKG_NAME)/Library/HdaLib/HdaLib.inf'
+            'HdaLib|Platform/$(BOARD_PKG_NAME)/Library/HdaLib/HdaLib.inf',
+            'VtdPmrLib|Silicon/CommonSocPkg/Library/VtdPmrLib/VtdPmrLib.inf'

```

Comment:
Better to have Platform code in separate commit if platform code does not use VtdPmrLib - SetDmaProtectedRange()/DisableDmaProtection().

Reply:
Yes, understood.  I just want to make it compile so that I know the library is fine. 
Without adding the library in build, it is hard to test that.
Also I will submit follow-on patches to enable DMA protection on APL platform.

Reply:
If you really want to remove this from BoardConfig.py, please let me know.

Reply:
I missed your commit message. No concern on this.

Reply:
Thx!  :)